elementary/clients/dbt/dbt_log.py

```diff
@@ -0,0 +1,38 @@
+
+from dataclasses import dataclass
+import json
+from typing import Iterator, Optional
+
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class DbtLog:
+    msg: Optional[str]
+    level: Optional[str]
+    exception: Optional[str]
+
+    @classmethod
+    def from_log_line(cls, log_line: str) -> 'DbtLog':
+        log = json.loads(log_line)
+        return cls(
```

Comment:
Can we add a method for `get_json` that returns `log.get("info") or log.get("data") or {}`?
Afterwards I'd call it once and read the properties from it.

Reply:
sadly it wont always work (for example in `level`)